Mathlib/CategoryTheory/Sites/Continuous.lean

```diff
@@ -0,0 +1,189 @@
+/-
+Copyright (c) 2024 Joël Riou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joël Riou, Andrew Yang
+-/
+import Mathlib.CategoryTheory.Sites.IsSheafOneHypercover
+
+/-!
+# Continuous functors between sites.
+
+We define the notion of continuous functor between sites: these are functors `F` such that
+the precomposition with `F.op` preserves sheaves of types (and actually sheaves in any
+category).
+
```

Comment:
It would be helpful to have somewhere a warning about the explicit universe parameter in the definition of `Functor.IsContinuous`. I.e. it would be more precise to say "`F.IsContinuous.{w}` means that precomposition with `F.op` preserves sheaves of `Type w` (and actually sheaves in any category `C` with `Category.{w} C`)", or something like that. 

(I know this comment doesn't really have anything to do with the main content of this PR, but I think it's worth improving documentation here.)

Reply:
Thanks very much for the reviews! Indeed, this PR is also motivated by an attempt to get q condition on Grothendieck topologies (the preservation of 1-hypercovers) which would imply `Functor.IsContinuous.{w}` for any `w`!

I have tried to add comments about these auxiliary universes in the docstring of the file.